heap_sort/c/heap.c

```diff
@@ -0,0 +1,61 @@
+#include "./heap.h"
+
```

Comment:
heap_sort/c/heap.c should include its header file heap_sort/c/heap.h  [build/include] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*